library/std/src/path.rs

```diff
@@ -2087,9 +2087,12 @@ impl Path {
     /// assert!(path.starts_with("/etc"));
     /// assert!(path.starts_with("/etc/"));
     /// assert!(path.starts_with("/etc/passwd"));
-    /// assert!(path.starts_with("/etc/passwd/"));
+    /// assert!(path.starts_with("/etc/passwd/")); // extra slash(es) is okay
```

Comment:
Are you sure this will work with more than one slash?
```suggestion
    /// assert!(path.starts_with("/etc/passwd/")); // extra slash is okay
```

Reply:
Yes, try `assert!(path.starts_with("/etc/passwd//"));`

Reply:
Ok, let's add that in the doc-tests then.
```suggestion
    /// assert!(path.starts_with("/etc/passwd/")); // extra slash is okay
    /// assert!(path.starts_with("/etc/passwd///")); // multiple extra slashes are okay
```

Reply:
Good idea